api/db/changelog/db.changelog-218-drop-unused-columns-user-table.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="thibault" id="db.changelog-218-drop-unused-columns-user-table">
+    <dropColumn columnName="compliance_training_bypass_time" tableName="user"/>
+    <dropColumn columnName="compliance_training_completion_time" tableName="user"/>
+    <dropColumn columnName="compliance_training_expiration_time" tableName="user"/>
+    <dropColumn columnName="data_use_agreement_bypass_time" tableName="user"/>
+    <dropColumn columnName="data_use_agreement_completion_time" tableName="user"/>
+    <dropColumn columnName="data_use_agreement_signed_version" tableName="user"/>
```

Comment:
The user Data Use Agreement is now called the Data User Code of Conduct, and the signed DUCC version is stored in the table `user_code_of_conduct_agreement`